### PR TITLE
Mention when to enable the 'editor' feature

### PR DESCRIPTION
**Pull request by rchriste**
I was confused at how to enable the Editor feature, which is to enable
the `editor` feature. Which is obvious in hindsight. This small doc PR
is meant to help prevent this point of confusion in the future.

### CODE REVIEW

File: inquire/src/lib.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@
 //!   - Custom extensions for files created by [`Editor`] prompts;
 //!   - and many others!
 //!
-//! \* Date-related features are available by enabling the `date` feature.
+//! \* Date-related features are available by enabling the `date` feature and enable the `editor` feature for the text editor features
```
Comment (mikaelmello) [inline inquire/src/lib.rs:31]:
```suggestion
//! * The Editor and DateSelect prompts are available by enabling the `editor` and `date` features, respectively.
```

I think this reads a little better now that we're mentioning both prompts, wdyt?

Comment (mikaelmello) [inline inquire/src/lib.rs:31]: Additionally, what do you think of moving this line to be placed _before_ `# Features`? It feels like the asterisks and the explanation are too far apart as well.

**Review by mikaelmello — COMMENTED** [review]
Makes a lot of sense, sorry for missing that lol.

Thanks for your PR!

File: inquire/src/lib.rs (hunk 1 above)
Comment (rchriste) [reply inquire/src/lib.rs:31]: I agree

Comment (rchriste) [reply inquire/src/lib.rs:31]: I agree that is better. I will update the PR.

Comment (rchriste) [reply inquire/src/lib.rs:33]: I moved it per your idea. Please let me know if this works.

**Review by rchriste — COMMENTED** [review]
Take @mikaelmello 's suggestion

File: inquire/src/lib.rs [hunk 2]
```diff
@@ -28,7 +30,7 @@
...
```
Comment (mikaelmello) [inline inquire/src/lib.rs:33]:
```suggestion
```

This line was accidentally kept I believe

**Comment by mikaelmello** [comment]
Not sure what the lint is complaining about, but it looks to be about LF to CRLF changes added

File: inquire/src/lib.rs (hunk 2 above)
Comment (rchriste) [reply inquire/src/lib.rs]: Thanks removed

**Comment by rchriste** [comment]
> Not sure what the lint is complaining about, but it looks to be about LF to CRLF changes added

The issue was an extra whitespace after the `//!`

**Review by mikaelmello — APPROVED** [review]

**Pull request merged**